test/index.js

```diff
@@ -10,39 +10,39 @@ describe('normalize', function() {
     var type = 'string';
     var value = 'test string';
     var result = normalize(type, value);
-    expect(result).toEqual(value);
+    expect(result).toBe(value);
```

Comment:
Ah, sorry, yes, that's what got me wondering about this -- sorry, I made this a while back :-)

I respectfully disagree about equals, preferring strict comparison where applicable -- I guess that touches at the core of our differing ideas here, in a way.

In any event, for the change I was going for it was quite essential (testing that the rejection signal was properly changed from `null` to `defined`). But if the contract will be that `null` and `undefined` should be treated interchangably by the users of this lib, equals would be fine of course.